### DISCUSSION TITLE
Header Text Styling

### DISCUSSION THREAD

**Discussion by psjnr84** [Clooos/Bubble-Card#957] [Questions about config, custom styles and templates] [answered]
Hi All,

Are you able to let me know if the Bubble Card styling supports the ability to justify the text within a button for example (alignment to the centre of the card instead of left justification)?

**Comment by Mystically11** [comment, marked as answer]
try this:

```
.bubble-name-container {
  align-items: center;
}
```

**Comment by orestkos** [comment]
It works, thank you :)
